no-kendoui-professional-in-package-manager.md

```diff
@@ -0,0 +1,67 @@
+
+---
+title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+description: Kendo UI Professional Is Not Available in the AppBuilder Package Manager When Using a DevCraft License
+type: Troubleshooting
+page_title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+slug: no-kendoui-professional-in-package-manager
+position: 0
+tags: devcraft, kendo, package manager, manage packages, appbuilder, license
+ticketid: 1017056, 1017115
+
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+## Description
+When having two subscriptions, the Devcraft subscription gives you the possibility to use the Kendo UI Professional. However, the Kendo UI Professional package cannot be installed in an AppBuilder app using the Package Manager.
+
+The Manage Packages window lets you install the Kendo UI Core package only. The Kendo UI Professional package is disabled, even though you have a valid Kendo UI Professional subscription.
```

Comment:
There is some redundancy in the two paragraphs. Consider simplifying and merging them into something like this:

Even though you have a DevCraft subscription plan which provides Kendo UI Professional, you cannot install the package using the AppBuilder Manage Packages wizard. The wizard only lets you install Kendo UI Professional.